pkg/relayer/config/types.go

```diff
@@ -132,6 +133,10 @@ type RelayMinerSupplierConfig struct {
 	// Other supplier types may embed other fields in the future. eg. "https" may
 	// embed a TLS config.
 	ServiceConfig *RelayMinerSupplierServiceConfig
+
+	// SigningKeyNames: a list of key names that can accept relays for that supplier.
```

Comment:
I still don't understand why it's a list. #PUC

Reply:
@Olshansk I pushed docusarus documentation update - please let me know if it's not enough and we should explain/repeat this in the code as well.